controller/src/beerocks/master/son_actions.cpp

```diff
@@ -154,14 +156,27 @@ void son_actions::unblock_sta(db &database, ieee1905_1::CmduMessageTx &cmdu_tx,
             continue;
         }
         Socket *sd = database.get_node_socket(hostap);
-        auto request =
-            message_com::create_vs_message<beerocks_message::cACTION_CONTROL_CLIENT_ALLOW_REQUEST>(
-                cmdu_tx);
-        if (request == nullptr) {
-            LOG(ERROR) << "Failed building ACTION_CONTROL_CLIENT_ALLOW_REQUEST message!";
+        if (!cmdu_tx.create(0,
+                            ieee1905_1::eMessageType::CLIENT_ASSOCIATION_CONTROL_REQUEST_MESSAGE)) {
```

Comment:
There are three places in the controller where very similar code is used to build the Client Association Control Request message. It might be worth moving that to a single function. Unfortunately, we don't have a good place for it...

(nitpick)